src/components/manga/ChapterList.tsx

```diff
@@ -110,11 +110,11 @@ const ChapterList: React.FC<IProps> = ({ mangaId }) => {
 
     const firstUnreadChapter = useMemo(
         () =>
-            visibleChapters
+            chapters
                 .slice()
                 .reverse()
-                .find((c) => c.read === false),
-        [visibleChapters],
+                .find((chapter) => chapter.read === false),
```

Comment:
```suggestion
                .find((chapter) => !chapter.read),
```
just a nitpick, since read value can only be true and false